packages/rum/src/browser/requestIdleCallBack.spec.ts

```diff
@@ -0,0 +1,55 @@
+import type { RumConfiguration, ViewCreatedEvent } from '@datadog/browser-rum-core'
+import { LifeCycle, LifeCycleEventType } from '@datadog/browser-rum-core'
+import type { TimeStamp } from '@datadog/browser-core'
+import { noop } from '@datadog/browser-core'
+import { mockRequestIdleCallback } from '@datadog/browser-core/test'
+import type { BrowserRecord } from '../types'
+import { addExperimentalFeatures, ExperimentalFeature } from '../../../core/src/tools/experimentalFeatures'
+import { createElementsScrollPositions, startFullSnapshots } from '../domain/record'
+import type { ShadowRootsController } from '../domain/record'
+
+describe('startFullSnapshots', () => {
```

Comment:
**💬 suggestion:** ‏this spec file should focus on `requestIdleCallback.ts`, and should not be related to full snapshots in any way.